emperor/support_files/js/shapes.js

```diff
@@ -12,14 +12,14 @@ define(['jquery', 'three'], function($, THREE) {
    *
    * Return a correctly sized geometry that matches the plotting space
    *
-   * @param {shapeName} string, one of 'Sphere', 'Cube', 'Cone', 'Icosahedron'
+   * @param {string} shapeName One of 'Sphere', 'Cube', 'Cone', 'Icosahedron'
    * or 'Cylinder'.
-   * @param {ranges} object, with two arrays of "max" and "min" values as
-   * generated by the DecompositionModel's dimensionRanges property;
+   * @param {Object} ranges Object with two arrays of "max" and "min" values as
+   * generated by the `DecompositionModel` dimensionRanges property
    *
-   * @return THREE.Geometry, returns the requested geometry object with a size
+   * @return {THREE.Geometry} The requested geometry object with a size
    * appropriate for the data presented on screen.
-   *
+   * @function getGeometry
```

Comment:
This block is missing a star at the beginning of the comment block, and has an extra star at the end of the comment block.